tests/test_endpoint.py

```diff
@@ -110,3 +105,10 @@ def test_server_webhook_custom_action_encoded_data_returns_200():
 
     assert events == [SlotSet("test", "bar")]
     assert response.status == 200
+
+
+# ENSURE THIS IS ALWAYS THE LAST TEST FOR OTHER TESTS TO RUN
+# for some reason, sys.exit is actually called here
```

Comment:
This is actually expected given this line that gets called as a result of this test: https://github.com/RasaHQ/rasa-sdk/blob/c1fec2a333dd01145e363ab4ff739eed88448f3a/rasa_sdk/executor.py#L266 
I would investigate more on why this breaks the pytest command if placed anywhere else in the test module, maybe other users of Sanic have encountered this as well?

Reply:
Yes, I did some investigation and couldn't find anything helpful.
It appeared the line was actually being executed by pytest, so any other thing afterwards isn't being run.

Not sure why it appeared with the updated Sanic version though and I couldn't find anything linking it to the issue.

Reply:
I'm wondering if it's not because of the incongruent version of sanic-testing with sanic? I'd test this again once you upgrade to sanic 22.12, that's the version you used for sanic-testing.

Reply:
I was hoping that was the case too and tried it after the upgrade but it still persists.

Reply:
There should be a package that manages the ordering, but I'm not sure the effort is worth it for now.

Reply:
Yeah definitely not worth the effort.

Reply:
I'd just replace the last line comment to say `call to sys.exit() terminates pytest process` or smth along those lines.

Reply:
Do we know if all the tests in the suite run actually, maybe we actually need to disable this and investigate later?

Reply:
Ah just checked the CI, all looks good.

Reply:
Comment updated.